editor/src/execution/feedback-action.ts

```diff
@@ -60,6 +64,10 @@ export class ExecutedFeedbackCommand extends Command {
   redo(context: CommandExecutionContext): SModelRoot {
     addCssClassToElements(this.executed, ExecutedFeedbackCommand.EXECUTED_CSS_CLASS);
     addCssClassToElements(this.failed, ExecutedFeedbackCommand.FAILED_CSS_CLASS);
+    removeCssClassOfElements(this.executed, ExecutedFeedbackCommand.LAST_EXECUTED_CSS_CLASS);
```

Comment:
Is this needed here or would it also be ok only in the else case?

Reply:
It is needed otherwise all executed elements will be marked as "last executed".